packages/muelu/doc/TutorialReadTheDocs/pages/quick_start.rst

```diff
@@ -0,0 +1,192 @@
+First one has to choose a problem. For this tutorial the right choice is the option 0 for the Laplace 2D problem on a :math:`50 \times 50` mesh.
+
+.. image:: pics/tut1_1.png
+  :width: 10cm
+
+Next one has to choose a xml file with the multigrid parameters. Choose option 2 and put in **s1_easy.xml** as filename for the xml file containing the xml parameters that are used for the multigrid method.
+
+.. note:
+
+  Please make sure that you enter a filename that actually exists on your hard disk!
+
+.. image:: pics/tut1_2.png
+  :width: 10cm
+
+The s1_easy.xml file has the following content
+
+.. literalinclude:: ../../../test/tutorial/s1_easy.xml
+  :language: xml
+
+As one can easily find from the xml parameters, a multigrid method with not more than 3 levels and a damped Jacobi method for level smoothing shall be used.
+Next, choose option 0 and run the example. That is, the linear system is created and iteratively solved both by a preconditioned CG method (from the AztecOO package) with a MueLu multigrid preconditioner and a standalone multigrid solver (again using MueLu) with the given multigrid parameters.
+
+.. image:: pics/tut1_3.png
+  :width: 10cm
+
+Note that the line **mpirun -np 2 MueLu_tutorial_laplace2d.exe -nx ...** is the command that is executed in the background. The default is 2 processors used.
+After pressing a key we are ready for a first analysis as it is stated by the green letters **Results up to date!**
+
+.. image:: pics/tut1_4.png
+  :width: 10cm
+
+.. note: 
+
+  If the results are not up to date always choose option 0 first to recalculate the results.
+
+To check the output select option 1. This should produce the following output on screen.
+
+.. warning:
+
+  Insert output
+
+.. note:
+
+  Depending on the number of lines in your terminal you may have to scroll up to the top of the file
+
+These lines give you some information about the setup process with some details on the aggregation process and the transfer operators. Note that for this example three levels are built: Level 0 for the finest level, level 1 as intermediate level and level 2 for the coarsest level. Then an overview of the different multigrid levels is given by
+
+.. warning:
+
+  Insert screen output
+
+One can see that a three level multigrid method is used with a direct solver on the coarsest level and Jacobi level smoothers on the fine and intermediate level. Furthermore some basic information is printed such as the operator complexity.
+
+In the end the CG convergence is printed when applying the generated multigrid method as preconditioner within a CG solver from the AztecOO package in Trilinos. The numbers give the relative residual after the corresponding number of iterations as well as the solution time in seconds.
+
+.. warning:
+
+  Insert screen output
```

Comment:
There are a few of these around. I could help fill some of these in if needed. It's worth noting that sometimes readthedocs will generate a warning textbox, as seen on [level smoothers](https://muelu-tutorial.readthedocs.io/en/latest/pages/level_smoothers.html), but sometimes it does not, like in the [quick start](https://muelu-tutorial.readthedocs.io/en/latest/pages/quick_start.html). It is not clear to me at the moment why this happens inconsistently.

Edit: I just realized this is likely a syntax subtlety with `.. warning:` vs `.. warning::`

Reply:
@GrahamBenHarper I fixed the syntax and replaced all `.. warning:` and `.. note:` with `.. warning::` and `.. note::`, repsectively. Now, they get rendered at least, but still have to be addressed.

Reply:
Thanks for doing this @mayrmt!